end-to-end-tests/tests/runtime/sidebar/sidebarAuth.spec.ts

```
@@ -56,9 +57,15 @@ test("Connect action in partner auth sidebar takes user to the Extension Console
 
   const consolePagePromise = context.waitForEvent("page");
 
-  await connectLink.click({
-    noWaitAfter: true,
-  });
+  try {
+    await connectLink.click();
+  } catch {
+    // There is a known issue with Edge where clicking external links in the sidebar will implicitly cause the sidebar to be
```

👍

---

One thing to consider is to only conditionally swallow the error if this test is being run in msedge.